app/views/static_pages/contributing.html.erb

```diff
@@ -11,6 +11,9 @@
           working on projects to improve Odin and are always looking for people who want to join our growing team
           of maintainers.
         </p>
+        <p class='max-w-prose mx-auto mt-4 prose prose-gray text-sm italic dark:prose-invert text-gray-500 dark:text-gray-400'>
+          If you are interested in supporting us financially, you can learn about that <%= link_to 'here', support_us_path %>
```

Comment:
Nit: this is non descriptive link text, which is something we are trying to get rid off. What about `on our page supporting page` or `on our page on supporting The Odin Project`?

Reply:
nice catch 🔥